### DISCUSSION TITLE
Syntax issue using x-show in a template

### DISCUSSION THREAD

**Discussion by tmcmillantx** [alpinejs/alpine#3085] [1. Help] [answered]
I am filtering a grid of logos based on a category that is contained in an array of data. I am having trouble getting the syntax correct for pulling the category (item.cat) into the x-show directive inside the template. Here is what I have that is not working-

`<div class="logo-img" x-show="tab === `item.cat` || tab === 'all'" x-transition.duration.500ms>`

Do I still include the quote marks? Do I need the backticks? I am sure it's a simple fix but I can't seem to get the right combo on the syntax.
Thanks for the assistance.

**Comment by SimoTod** [comment, marked as answer]
The syntax is correct. From your snipoet is not clear what the data is though. Are you sure that your `tab` value matches item.cat?

**Reply by tmcmillantx** [reply]
The issue is that the category from the data is not rendering in the html. It renders as item.cat instead of the value for item.cat.
Screenshot of rendered html-
![image](https://user-images.githubusercontent.com/10051314/182960057-06392a0c-a801-4ca3-b312-1f3e8bb106b3.png)
Data-
![image](https://user-images.githubusercontent.com/10051314/182960264-836fb3d0-b329-469e-87ee-021b058dc058.png)

**Reply by SimoTod** [reply]
Item.cat is a variable. You don't get to see the value in the html. Can you post a full example to illustrate the issue?

**Reply by tmcmillantx** [reply]
Ok, that's good to know. I didn't realize that. I have the code working now. Thanks for the assistance.
